packages/calypso-products/src/constants/jetpack.ts

```diff
@@ -736,6 +748,11 @@ export const JETPACK_RESET_PLANS_BY_TERM = < const >[
 		yearly: PLAN_JETPACK_STARTER_YEARLY,
 		monthly: PLAN_JETPACK_STARTER_MONTHLY,
 	},
+	{
+		biYearly: PLAN_JETPACK_GROWTH_BI_YEARLY,
+		yearly: PLAN_JETPACK_GROWTH_YEARLY,
+		monthly: PLAN_JETPACK_GROWTH_MONTHLY,
+	},
 ];
 export const JETPACK_PLANS = < const >[
 	PLAN_JETPACK_FREE,
```

Comment:
There are a few places I see where we could add growth, particularly the upgrade paths where we'd show an upgrade from Growth -> Complete. I'll get those added before merging